src/attr/_make.py

```diff
@@ -2277,7 +2310,7 @@ def fmt_setter_with_converter(
             names_for_globals[val_name] = a.validator
             names_for_globals[attr_name] = a
 
-    if post_init:
+    if post_init and not attrs_init:
```

Comment:
So this is a valid decision, but could you explain why you decided for it? I think I'd tend to leave it inside of `__attrs_init__` but am happy to be convinced otherwise.

Reply:
As this code is written, if init=True, the user will end up writing an `__init__` like 

```
def __init__(self, *args, **kwargs):
     ... # pre-init stuff
     self.__attrs_init__(*args, **kwargs)
     self.__attrs_post_init__()
```
In theory this allows the user to do something in between `__attrs_init__` and `__attrs_post_init__`, but that doesn't make a lot of sense since they can just do that in `__attrs_post_init__`. It's probably worse to do it like this because users will forget to call post-init. 

I will change this back to be called by `__attrs__init__`.

Reply:
OK. I'm all for leaving doors open, but as you say: we don't lose anything here. And I already smell the confused bug reports about post init not being called…